scripts/ci/qemu-run.sh

```diff
@@ -55,6 +55,7 @@ This is the QEMU wrapper script for the Dasharo Open Source Firmware Validation.
     DRIVE_PATH  path to file which will be added as a virtual hard drive. The file
                 should match formats described in chapter Disk Images of QEMU
                 documentation
+    QCOW2_DRIVE path to .qcow2 file which will be connected as block device
```

Comment:
How is this different from `DRIVE_PATH`? Cannot we use just that? It still shows up in `Add Boot Option` in BIOS.

Reply:
Interesting, thought Qemu needs specific driver to be specified. But you are right `DRIVE_PATH` works fine too. I have deleted this commit.